google-cloud-secret_manager/samples/acceptance/snippets_test.rb

```diff
@@ -298,4 +298,19 @@
       }.must_output(/Updated secret/)
     end
   end
+  describe "#update_secret_with_alias" do
```

Comment:
Nit: Can you insert a blank line above this line, so the `describe` blocks are separated vertically?